### PR TITLE
CI: Use buildroot toolchain for building

### PR DESCRIPTION
**Pull request by tuxuser**
* Use webOS buildroot toolchain: https://github.com/openlgtv/buildroot-nc4/
   - Brings down minimum required glibc to v2.12.2
* For compatibility with glibc 2.12.2: Link `librt` explicitly
* Forward `CMAKE_TOOLCHAIN_FILE` cmake parameter to `libyuv` build

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@ name: Build
 on:
   push:
 
+env:
+  TOOLCHAIN_URL: https://github.com/openlgtv/buildroot-nc4/releases/latest/download/arm-webos-linux-gnueabi_sdk-buildroot.tar.gz
```
Comment (Informatic) [inline .github/workflows/build.yml:7]: I'd rather have this pinned to a specific version (and possibly with sha256sum check too)

**Review by Informatic — CHANGES_REQUESTED** [review]

File: libyuv/CMakeLists.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ include(ExternalProject)
 ExternalProject_Add(
   libyuv
   URL https://chromium.googlesource.com/libyuv/libyuv/+archive/fdc71956bdade0012b0628b5011b567c06c72308.tar.gz
```
Comment (Informatic) [inline libyuv/CMakeLists.txt:5]: I need to retest this personally, but the last time I tried this CMake was unable to download a dependency over TLS due to missing `https` support in libcurl. I assume this has already been fixed in latest toolchain release.

Comment (tuxuser) [reply libyuv/CMakeLists.txt:5]: yeah, no such issue occured when trying with the buildroot toolchain at least.

File: .github/workflows/build.yml (hunk 1 above)
Comment (tuxuser) [reply .github/workflows/build.yml:7]: Ok, sounds reasonable, will implement that.

**Comment by tuxuser** [comment]
* Changed toolchain tarball to versioned URL
* Verify sha256 hash of downloaded tarball

**Review by Informatic — APPROVED** [review]

**Pull request merged**